app/routes/settings/email-preferences.js

```diff
@@ -8,5 +8,10 @@ export default Route.extend({
   },
   model() {
     return this.get('authManager.currentUser').query('emailNotifications', { include: 'event' });
+  },
+  actions: {
+    willTransition() {
+      this.get('authManager.currentUser').rollbackAttributes();
```

Comment:
@niranjan94 I had a question. There is no such thing as 'save' button at this route. Its just the checkbox states. They will be either on or off. What do we rollback then?